internal/controllers/image/upload.go

```diff
@@ -43,76 +45,78 @@ func (r *orcImageReconciler) hashVerifier(ctx context.Context, orcImage *orcv1al
 		} else {
 			log.V(logging.Info).Info("download hash verification failed", "expected", expectedValue, "got", hash)
 			msg := "download hash verification failed. got: " + hash
-			r.recorder.Eventf(orcImage, corev1.EventTypeWarning, "HashVerificationFailed", msg)
+			// actuator.recorder.Eventf(orcImage, corev1.EventTypeWarning, "HashVerificationFailed", msg)
```

Comment:
In general we should be generating events, and this should almost certainly be one of them. However, as we're not currently doing that consistently I think it's ok to remove this one until we fix it consistently everywhere.